packages/react/src/patterns/blocks/ContentGroupSimple/__stories__/ContentGroupSimple.stories.js

```diff
@@ -0,0 +1,76 @@
+import './index.scss';
+import { text, object, withKnobs, select } from '@storybook/addon-knobs';
+import ContentGroupSimple from '../ContentGroupSimple';
+import React from 'react';
+// import readme from '../README.md';
+import { storiesOf } from '@storybook/react';
+
+storiesOf('Patterns (Blocks)|Content Group Simple', module)
+  .addDecorator(withKnobs)
+  //   .addParameters({
+  //     readme: {
+  //       sidebar: readme,
+  //     },
+  //   })
```

Comment:
Need to uncomment this?

Reply:
I've missed that, It will be corrected on the latest version.